packages/web/src/components/pages/Tests/TestDetails/TestSettings/SettingsVariables/Arguments.tsx

```diff
@@ -28,6 +28,19 @@ import {prettifyArguments} from '@utils/prettifyArguments';
 
 import {ArgumentsWrapper} from './Arguments.styled';
 
+const formatArgsArray = (args: string[]) => {
+  return args.map(arg => {
+    const trimmedArg = arg.replace(/\s+|(?<=[^'])['"](?=[^'])/g, match => {
+      if (match.trim() === '') {
+        return ' '; // Replaces multiple spaces with a single space
+      }
+      return `\\${match}`; // Escapes quotes
```

Comment:
It would work, although won't display nice. It will unnecessarily escape some characters.

As an example `"abc'def"` will become `"abc\'def"` despite, `\` is redundant here